### ISSUE TITLE
STN Randomly Opens

### ISSUE THREAD

**Issue by BreinLouz** [yuliskov/SmartTubeNext#1118]
STN 13.60, Shield Pro 2019

STN will open semmingly randomly in a middle of a movie on Plex, or a game on NFLGamePass (Not tried with other apps).
No button pressed whatsoever.

**Comment by Virusmater** [comment]
Happened the same for me. Same version, same Shield.

**Comment by Virusmater** [comment]
Happens after casting from iPhone. Is there any way to help resolve this? Collect logs or something

**Comment by northernpowerhouse** [comment]
I also have this issue, at least with plex playback, nvidia shield 2019, with the beta release installed.

**Comment by northernpowerhouse** [comment]
I investigated further and only have this issue after casting, turns out the youtube/vanced app was still 'connected'. Going back into the casting app and hitting disconnect in the cast menu solved the problem for me.

**Comment by hogi1200** [comment]
Same issue for me.
STN beta 13.81, Sony Android TV

Luckily I found this issue reported here. Will monitor from now regarding connected YouTube apps.

STN was opening randomly while using other apps on TV without any intentional trigger. 
And i think one time the TV even turned itself on just to show STN app.

**Comment by BreinLouz** [comment]
> I investigated further and only have this issue after casting, turns out the youtube/vanced app was still 'connected'. Going back into the casting app and hitting disconnect in the cast menu solved the problem for me.

Indeed, identified the same cause and all devices need to be disconnected for this issue to stop.
Suggesting to fix.

**Comment by jaimehrubiks** [comment]
No other solution?

**Comment by marnixbent** [comment]
Same issue here with the latest version of this date, have to disconnect all casting devices to make it stop
